src/more-infos/more-info-media_player.html

```diff
@@ -64,6 +72,16 @@
           on-change='volumeSliderChanged' class='flex'>
         </paper-slider>
       </div>
+      <div class='controls layout horizontal justified' hidden$='[[!supportsSelectSource]]'>
+          <iron-icon class="source-input" icon="mdi:login-variant"></iron-icon>
+          <paper-dropdown-menu class="source-input" no-label-float label='[[source]]'>
```

Comment:
We shouldn't use the label to show the current selected option. Instead it should say 'Source'.